app/src/componentLibrary/dropdown/dropdownOption/dropdownOption.jsx

```diff
@@ -41,12 +46,15 @@ export const DropdownOption = React.forwardRef((props, ref) => {
         hidden,
         hover: highlightHovered,
       })}
-      title={(disabled && title) || undefined}
+      title={((disabled || isTextTruncated) && title) || undefined}
```

Comment:
Let's not complicate this place.
I suppose it will be ok to set `title` permanently as it was done for similar case in https://jiraeu.epam.com/browse/EPMRPP-85818 (https://github.com/reportportal/service-ui/pull/3629/files).